packages/manager/cypress/e2e/core/kubernetes/lke-update.spec.ts

```diff
@@ -215,6 +216,61 @@ describe('LKE cluster updates', () => {
     ui.toast.findByMessage('Recycle started successfully.');
   });
 
+  it('can upgrade the kubernetes version from the landing page', () => {
+    const oldVersion = '1.25';
+    const newVersion = '1.26';
+
+    const cluster = kubernetesClusterFactory.build({
+      k8s_version: oldVersion,
+    });
+
+    const updatedCluster = { ...cluster, k8s_version: newVersion };
+
+    mockGetClusters([cluster]).as('getClusters');
+    mockGetKubernetesVersions([newVersion, oldVersion]).as('getVersions');
+    mockUpdateCluster(cluster.id, updatedCluster).as('updateCluster');
+    mockRecycleAllNodes(cluster.id).as('recycleAllNodes');
+
+    cy.visitWithLogin(`/kubernetes/clusters`);
+
+    cy.wait(['@getClusters', '@getVersions']);
+
+    cy.findByText(oldVersion).should('be.visible');
+
+    cy.findByText('UPGRADE').should('be.visible').should('be.enabled').click();
+
+    ui.dialog
+      .findByTitle(
+        `Step 1: Upgrade ${cluster.label} to Kubernetes ${newVersion}`
+      )
+      .should('be.visible');
+
+    ui.button
+      .findByTitle('Upgrade Version')
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    mockGetClusters([updatedCluster]).as('getClusters');
+    cy.wait(['@updateCluster', '@getClusters']);
```

Comment:
```suggestion
    mockGetClusters([updatedCluster]).as('getClusters');

    ui.button
      .findByTitle('Upgrade Version')
      .should('be.visible')
      .should('be.enabled')
      .click();

    cy.wait(['@updateCluster', '@getClusters']);
```

This appears to be what we actually need here to get the version to update to 1.26. I am not 100% clear on why the mock needs to go before the user action, but know that often when the right requests are being made, but the right data isn't showing, it's this kind of (timing?) issue.

Reply:
glad I wasn't going crazy! I'm also curious to how this test managed to pass in the automatic test checks when it's been failing for us.. 😅

Reply:
Also seeing this test fail locally.

Reply:
Good catch @mjac0bs 

I knew something felt weird when I wrote that 

Fixed in 1d66657